docs/guides/es2k/enabling-comms-channel.md

```diff
@@ -0,0 +1,132 @@
+# Enable COMMS channel
+
+COMMS channel enablement is required to establish communication between
+different compute complexes on IPU. Here, we will be giving an example
+to enable communication between ACC and Host. This allows P4Runtime client
+`p4rt-ctl` running on Host to communicate with `infrap4d` running on ACC.
+
+Port used for COMMS channels are defined by the node policy on IMC.
+
+## 1. Interrupt default start-up routine
+
+Reboot IMC and type ``N`` when the following message is shown on IMC console::
+
+```text
+start ipumgmtd and auxiliary script [Y/N] \
+(default start both automatically after 10 seconds)?
+```
+
+## 2. Specify COMMS channel ports
+
+Modify `/etc/dpcp/cfg/cp_init.cfg` to change the value of `comm_vports`.
+
+```text
+/* ACC_LAN_APF_VPORT_3 <--> HOST0_LAN_APF_VPORT_3 */
+comm_vports = (([5,0],[4,0]),([0,3],[4,2]));
+```
+
+This will enable communication between Host and ACC.
+
+## 3. Start the IMC
+
+Run the IMC start-up script.
+
+```bash
+root@mev-imc:~# /etc/init.d/run_default_init_app
+```
+
+By default, `cpf_host` parameter in `/etc/dpcp/cp_init.cfg` is set to 4 which
+enables ACC. If the start-up script is executed successfully, ACC comes up with a
+statically assigned IP address `192.168.0.2` to the eth0 network interface.
+You can access ACC from IMC over an SSH session using this IP address.
+
+## 3. Load the IDPF driver on Host
+
+```bash
+modprobe idpf
```

Comment:
should we mention compatable ATE kernel should be installed?
to get this driver on HOST

Reply:
This will be part of IPU SDK bringup document